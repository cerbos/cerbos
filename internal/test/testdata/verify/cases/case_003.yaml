description: Valid case with different ipAddress attributes to test variable evaluation
