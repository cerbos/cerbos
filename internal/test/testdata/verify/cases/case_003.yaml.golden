{
  "suites":  [
    {
      "file":  "udf_test.yaml",
      "name":  "TestSuite",
      "principals":  [
        {
          "name":  "bev",
          "resources":  [
            {
              "name":  "pending_leave_request",
              "actions":  [
                {
                  "name":  "delete",
                  "details":  {
                    "result":  "RESULT_PASSED"
                  }
                },
                {
                  "name":  "approve",
                  "details":  {
                    "result":  "RESULT_PASSED"
                  }
                }
              ]
            }
          ]
        },
        {
          "name":  "matt",
          "resources":  [
            {
              "name":  "pending_leave_request",
              "actions":  [
                {
                  "name":  "delete",
                  "details":  {
                    "result":  "RESULT_PASSED"
                  }
                },
                {
                  "name":  "approve",
                  "details":  {
                    "result":  "RESULT_PASSED"
                  }
                }
              ]
            }
          ]
        }
      ],
      "summary":  {
        "overallResult":  "RESULT_PASSED",
        "testsCount":  4,
        "resultCounts":  [
          {
            "result":  "RESULT_PASSED",
            "count":  4
          }
        ]
      }
    }
  ],
  "summary":  {
    "overallResult":  "RESULT_PASSED",
    "testsCount":  4,
    "resultCounts":  [
      {
        "result":  "RESULT_PASSED",
        "count":  4
      }
    ]
  }
}